webvtt.html

```diff
@@ -1171,7 +1168,7 @@
     contain the substring "<code>--></code>" (U+002D HYPHEN-MINUS, U+002D HYPHEN-MINUS, U+003E
     GREATER-THAN SIGN).</p>
 
-    <p>A <dfn>WebVTT cue</dfn> consists of the following components, in the given order:</p>
```

Comment:
E.g. this paragraph:
"Each media element has a list of newly introduced cues, which must be initially empty. Whenever a text track cue is added to the list of cues of a text track that is in the list of text tracks for a media element, that cue must be added to the media element's list of newly introduced cues. Whenever a text track is added to the list of text tracks for a media element, all of the cues in that text track's list of cues must be added to the media element's list of newly introduced cues. When a media element's list of newly introduced cues has new cues added while the media element's show poster flag is not set, then the user agent must run the time marches on steps."

But I can live with "webvtt cue box". I'll rebase this and then we can try again.